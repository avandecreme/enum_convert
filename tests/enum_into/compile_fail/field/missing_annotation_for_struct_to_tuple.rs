use enum_convert::EnumInto;

#[derive(EnumInto)]
#[enum_into(Target)]
enum Source {
    #[enum_into(Target::Tuple)]
    Struct {
        // #[enum_into(Target::Tuple.1)] missing
        aa: i32,
        #[enum_into(Target::Tuple.0)]
        bb: i32,
    },
}

enum Target {
    Tuple(i64),
}

fn main() {}
