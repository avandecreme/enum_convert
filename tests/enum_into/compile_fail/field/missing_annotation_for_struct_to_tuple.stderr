error: Missing required mapping to named field for Target::Tuple
 --> tests/enum_into/compile_fail/field/missing_annotation_for_struct_to_tuple.rs:9:9
  |
9 |         aa: i32,
  |         ^^
